tests/pytests/test_debug_commands.py

```diff
@@ -182,3 +182,97 @@ def testNumericIndexSummaryWrongArity(self):
 
     def testDumpSuffixWrongArity(self):
         self.env.expect('FT.DEBUG', 'DUMP_SUFFIX_TRIE', 'idx1', 'no_suffix').raiseError()
+
+def get_and_check_threadpools_count(env, threadpools_dict, expected_len):
+    threadpools_titles = threadpools_dict[0::2] if env.protocol == 2 else list(threadpools_dict.keys())
+    env.assertEqual(len(threadpools_titles), expected_len)
+    return threadpools_titles
+    
+def check_threads_count(env, threadpools_dict, expected_len, thpool_pos):
+    if env.protocol == 2:
+        # In resp2 for each thpool the output is in the format of [thpool_title, [thread_bt_#0, thread_bt#1....]]
+        resp2_thpool_output_len = 2
+        curr_backtraces = thpool_pos * resp2_thpool_output_len + 1 # bt relative position 
+        backtraces_titles = threadpools_dict[curr_backtraces][0::2]
+    elif env.protocol == 3:
+        backtraces_titles = threadpools_dict[thpool_pos].keys()
+    env.assertEqual(expected_len, len(backtraces_titles))
+    
+def threadpool_name_to_title(thpool_name):
+    return f"=== {thpool_name} THREADS BACKTRACE: ==="
+
+def threadpool_title_to_name(thpool_title):
+    if thpool_title == "=== GC THREADS BACKTRACE: ===":
+        return "GC"
+    elif thpool_title == "=== WORKERS THREADS BACKTRACE: ===":
+        return "WORKERS"
+    elif thpool_title == "=== CLEANSPEC THREADS BACKTRACE: ===":
+        return "CLEANSPEC"
+    else:
+        return None  
+        
+def DumpBacktrace_ALL(env: Env, threadpools_attr):
+    # Ask for all threadpools
+    threadpools_dict = env.cmd('FT.DEBUG', 'DUMP_THREADPOOL_BACKTRACE', 'ALL')
+    threadpools_titles = get_and_check_threadpools_count(env, threadpools_dict, len(threadpools_attr))
+    
+    # Ensure that all the threadpools appear only once
+    for i, threadpool in enumerate(threadpools_titles):
+        thpool_name = threadpool_title_to_name(threadpool)
+        if thpool_name is None:
+            env.assertTrue(False, message=(f"Threadpool title {threadpool} is unexpected"))
+            continue
+        env.assertEqual(threadpools_attr[thpool_name]["status"], 'NOT_FOUND')
+        threadpools_attr[thpool_name]["status"] = 'FOUND'
+        check_threads_count(env, 
+                            threadpools_dict, 
+                            expected_len= threadpools_attr[thpool_name]["threads_count"], 
+                            thpool_pos = i if env.protocol == 2 else threadpool)
+    
+
+        
+def DumpBacktrace(protocol):
+    if MT_BUILD:
+        WORKER_THREADS = 3
+        expected_threadpools_cnt = 3
+        module_args= f'WORKER_THREADS {WORKER_THREADS} MT_MODE MT_MODE_FULL'
+    else:
+        expected_threadpools_cnt = 2
+        module_args=''
+        
+    env = Env(protocol=protocol, moduleArgs=module_args)
+    env.skipOnCluster()
```

Comment:
On cluster, each shard writes this info to its crash log, and each shard can be connected and be invoked with the command `FT.DEBUG DUMP_THREADPOOL_BACKTRACE ...` ?

Reply:
The coordinator doesn't support ft.debug, no?

Reply:
Right, we need to send `_FT.DEBUG` to each shard we want to inspect. So this is no different.